.vsts-ci-official.yml

```diff
@@ -0,0 +1,183 @@
+variables:
+  - name: _TeamName
+    value: DotNetCore
+  - name: _DotNetArtifactsCategory
+    value: .NETCore
+  - name: Codeql.Enabled
+    value: true
+
+# CI and PR triggers
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+pr:
```

Comment:
Are we running this pipeline on pull requests? Do we need the `pr:` section?

Reply:
In general, for the migration, I didn't simplify the pipelines, that can be done in a follow up step.

Reply:
Note that we would want to simplify both `.vsts-ci.yml` and `.vsts-ci-official.yml` as both are written for public/internal but they run only in one context now. But I would be hesitant simplifying them now - I think in the future we will want to have even the public pipelines using 1ES template, right? Then we could reuse this pipeline again for both public and internal runs.